server_admin/topics/identity-broker/first-login-flow.adoc

```diff
@@ -45,8 +45,8 @@ Create User If Unique::
   If it's not, then the authenticator just creates a new local {project_name} account and links it with the identity provider and the whole flow is finished.
   Otherwise it goes to the next `Handle Existing Account` subflow.
   If you always want to ensure that there is no duplicated account, you can mark this authenticator as `REQUIRED`. In this case, the user
-  will see the error page if there is an existing {project_name} account and the user will need to link his identity provider account through Account management.
-
+  will see the error page if there is an existing {project_name} account and the user will need to link his identity provider account through Account management. 
+  
```

Comment:
Is it possible to add one more NOTE to this and mention, that user can just replace this authenticator with the `Detect Existing Broker User`  to skip creation of users? Something like this:
```
NOTE: If you want to skip the ability to create new users, but you want that users authenticated from identity provider 
must already exists in ${project_name} with same username or email like the user from identity provider, you can create new flow and replace `Create User If Exists` authenticator with
`Detect Existing Broker User` . More details in the examples below.
```

Reply:
Done. I have added a link to the "Detect Existing User First Login Flow" section.